docs/1.0.0-ANNOUNCEMENT.md

```diff
@@ -11,7 +10,7 @@ stable 1.0.0 release scheduled for January 2022. The release *will* include:
   for the legacy client updater
 
 As discussed in [ADR 2](https://github.com/theupdateframework/python-tuf/blob/develop/docs/adr/0002-pre-1-0-deprecation-strategy.md), this
-release *will not* include any legacy code, as its maintenance has become
+release *does not* include any legacy code, as its maintenance has become
 infeasible for the python-tuf team. The pre-1.0.0 deprecation strategy from ADR
 2 applies as follows:
```

Comment:
Now that this code has been removed, should this statement be moved down? The next paragraph seems more important.